cli/pkg/providers/gcp/gcp.go

```diff
@@ -0,0 +1,88 @@
+package gcp
+
+import (
+	"cli/pkg/console"
+	"cli/pkg/workspace"
+	tmpl "cli/pkg/workspace/template"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/airyhq/airy/lib/go/tools"
+
+	getter "github.com/hashicorp/go-getter"
+	"gopkg.in/segmentio/analytics-go.v3"
+)
+
+type provider struct {
+	w         io.Writer
+	analytics console.AiryAnalytics
+}
+
+func New(w io.Writer, analytics *console.AiryAnalytics) *provider {
+	return &provider{
+		w:         w,
+		analytics: *analytics,
+	}
+}
+
+func (p *provider) GetOverrides() tmpl.Variables {
+	return tmpl.Variables{
+		NgrokEnabled: true,
```

Comment:
This is only needed for Minikube, but not in cloud environment. This should solve the problem with ingress and the ports.